src/Core/Domain/Product/Customization/Command/AddCustomizationFieldCommand.php

```diff
@@ -33,6 +33,8 @@
 
 /**
  * Adds customization field to a product
+ *
+ * @todo: this command along with update/deleteCustimizationField can be removed, because we only have a usecase for Set & RemoveAll
```

Comment:
Since https://github.com/PrestaShop/PrestaShop/pull/21260 has been merged I think you can rebase this one

Reply:
This one can be removed no?